src/poliastro/twobody/perturbations.py

```diff
@@ -76,3 +78,23 @@ def atmospheric_drag(t0, state, k, R, C_D, A, m, H0, rho0):
     rho = rho0 * np.exp(-(H - R) / H0)
 
     return -(1.0 / 2.0) * rho * B * v * v_vec
+
+
+def third_body(t0, state, k, k_third, third_body):
+    """Calculates 3rd body acceleration (km/s2)
+
+    Parameters
+    ----------
+    t0 : float
+        Current time (s)
+    state : numpy.ndarray
+        Six component state vector [x, y, z, vx, vy, vz] (km, km/s).
+    k : float
+        gravitational constant, (km^3/s^2)
+    third_body: ~OdeSolution object
```

Comment:
Why it has to be an `OdeSolution` object? Any callable that receives a `t0` (time elapsed since start of the propagation) and returns a state vector should work, right?